webTest/js/script.js

```diff
@@ -0,0 +1,70 @@
+// function hideShow(){
+//
+//   if(cheker%2=0){
+//     var hider=document.querySelector("#hide");
+//     hider.style.display="none";
+//   }
+// }
+var cheker = 2;
+var clikeShowHide = document.querySelector("#showHide");
+var hider = document.querySelector(".showOne");
+clikeShowHide.addEventListener('click', function() {
+    if (cheker % 2 === 0) {
```

Comment:
this logic works but couldn't it be simpler? just a boolean that switches between true and false

```
var isVisible = false;

// then in the handler, just set isVisible to switch, so
isVisible = !isVisible;
// that basically makes it alternate, if it was true, it will become false and vice versa, then you don't the **magic number 2**
```

Also it's better to just create a class in css called `.hidden { display: none }` and add/remove that class based on `isVisible`

I would also use .replace() to remove the class instead of concatenating.

Once you have that solution running, think about another way to get rid of the variable `cheker` or `isVisible` altogether - remember that this is `global state` and global state is very bad. `hint: you can just check if element.className contains *hidden* or not, to decide if it is visible or not)

let me know if these comments are confusing and I can go through them in more details